react/week2/react-w2/src/App.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
```

Comment:
Really nice solution here, all your functions are easy to follow and you use good naming and some good abstractions into components 👏 Be careful not to use too many abstractions if it's not really necessary. 
A good practice would be to put each of your components into their own files, e.g. a TodoList.js file. As your app grows, it is important to think about how you can structure the components into files, folders and subfolders to keep a clean code structure.

Reply:
Thank you for your feedback . Will keep all your points in mind.